Home: Small fixes

machikoyasuda opened the pull request:
closes #352 

- Removes the background color links
- Updates the images

thekaveman commented on src/index.html:
```diff
@@ -52,23 +57,17 @@ <h2 class="text-center text-md-start mt-5 mb-4">Bringing industry standards to C
-        target="_blank"
-        class="green-link fw-bold"
-        href="https://dot.ca.gov/ ">California Department of Transportation</a>
+      <a rel="noreferrer" target="_blank" class="fw-bold" href="https://dot.ca.gov/ ">California Department of Transportation</a>
```
I know this isn't part of this PR, but I just noticed this `href` has an extra space at the end, before the end quote.

machikoyasuda replied:
Fixed: https://github.com/cal-itp/calitp.org/pull/361/commits/2b626f1e7c65beb7f4f247b2f016688755080c74

thekaveman reviewed (commented):
You should be able to just `rebase` out these 2 commits: 2939777a16d87541fb17d064ac88cfdd70f27d44 and 7155093f6964869b3f4a440eb8ed43fc838bd544

ohmegasquared commented:
Links look great! No more drop shadow looks good. 

Was updating the images the dot images?
1. the first dots are flipped horizontally
<img width="625" alt="Screenshot 2024-07-16 at 12 09 59 PM" src="https://github.com/user-attachments/assets/0fc7cc9f-92da-46fc-9eeb-6bd2bea6bde0">

2. above our initiatives should be the straight line of dots
<img width="704" alt="Screenshot 2024-07-16 at 12 13 39 PM" src="https://github.com/user-attachments/assets/a497eaba-ab24-4b0e-905c-1b37f4516896">

3. the blue card backgrounds on the CTAs look good. Can you re-export the images, the stroke was updated to primary dark blue.  And are these cards 4 columns? they feel a little tight. 

(Probably a diff PR but wanted to call out -- I also removed "our" from the link for Press Page so it reads a bit easier)

machikoyasuda commented:
@ohmegasquared 

- Updated the mirror flipped first dot illustration.
- Added in the straight line of dots illustration.
- Removed the `our` from the Press page link.
- Alignment:

6-columns centered
<img width="1512" alt="image" src="https://github.com/user-attachments/assets/73821d2a-a51d-4403-b032-d5406308d3fe">
 
8-columns centered (with the image in its own separate column)
<img width="1512" alt="image" src="https://github.com/user-attachments/assets/0ea4760f-bf17-4f43-8aa2-ed0d94fff713">

I just noticed on Figma though... the Initatives section text is 6 columns, not 8. Should I change it (I can do it in a separate PR)?
<img width="206" alt="image" src="https://github.com/user-attachments/assets/7ced31fc-0bea-4042-a729-305fb3ca0f90">

thekaveman reviewed (approved):
Looks great! So much cleaner and more lively than the current homepage.

ohmegasquared commented:
I think the spacing for the initiatives looks good! 

In figma, the bringing industry standards and goals section looks wider with 6 columns than netlify, do we have strong margins/padding for that section? the container appears to be smaller 

the second divider seems to be a little streched? the circles look ovals to me (not sure if it is the old 2023 divider and just need to pull the new image?) 
<img width="445" alt="Screenshot 2024-07-22 at 11 25 01 AM" src="https://github.com/user-attachments/assets/22f1a0e5-c9cd-4d85-9e01-a0c26e548cca">

The hero section should spread 9 columns 
- 5 columns for text
- 4 col for the image
